ui/src/views/Rules.vue

```diff
@@ -219,6 +220,52 @@
               </div>
             </div>
           </div>
+          <div v-if="r.type === 'separator'" v-show="searchString.length == 0" class="list-view-pf-checkbox drag-here">
```

Comment:
```suggestion
          <template v-if="r.type === 'separator'">
            <div v-show="searchString.length == 0" class="list-view-pf-checkbox drag-here">
```
The `div`s having `v-if="r.type === 'separator'"` can be enclosed inside a `<template v-if="r.type === 'separator'">`. Then the `v-if` for the `div`s can be removed.

https://vuejs.org/v2/guide/conditional.html#Conditional-Groups-with-v-if-on-lt-template-gt